src/styles/global/layout.scss

```diff
@@ -50,11 +50,7 @@ main,
 }
 
 // Avoids double scroll issue on the page when overlay is open
-#layout.no-scroll {
-  position: fixed;
-  top: 0;
-  right: 0;
-  left: 0;
+.no-scroll {
   overflow: hidden;
```

Comment:
This doesn't work very well on iOS safari :(
I remember trying a few approaches to this problem before.
Have you tried something with `touch-action:none`?

https://user-images.githubusercontent.com/3356699/160032184-fa95c50d-b384-4420-b41b-b1f1a00c738e.mov

Reply:
Nice catch! I'll look into new approaches to make it works on iOS Safari.

Reply:
It is still not working [properly](https://sfj-e43e031--base.preview.vtex.app/) 😞 Let's use another approach for that.

Reply:
I tried a similar [approach](https://github.com/vtex-sites/base.store/pull/443 ) with position fixed. It[ locks the scroll inside the modal,](https://preview-443--base.preview.vtex.app/) but we always get the scroll position back to the top. Should we try to solve it using some external library? 🤔

Reply:
I tried a similar [approach](https://github.com/vtex-sites/base.store/pull/443 ) with position fixed. It[ locks the scroll inside the modal,](https://preview-443--base.preview.vtex.app/) but we always get the scroll position back to the top. Should we try to solve it using some external library? 🤔

Reply:
well, the solution above solves the bug when touching the `overlay`. But the bug remains while scrolling into the `minicart` just using iOS Safari. I also have tried different approaches to that problem but without success.

As this is a very specific known problem only on iOS Safari. I think we can go ahead. What do you think?